data/MI/incentives.json

```diff
@@ -1084,34 +1001,12 @@
       "homeowner"
     ],
     "short_description": {
-      "en": "$900 rebate per outdoor unit for mini/multi split air-source heat pumps. Limit 2 units."
+      "en": "$900 rebate per outdoor unit for mini/multi split air-source heat pumps. Limit 2 units. Additional $100 per indoor unit."
```

Comment:
combined MI-52 into MI-51. 51 was the additional $100 per indoor unit but I think they can be combined?